<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>701</width>
    <height>288</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="editInput">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>451</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="editOutput">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>150</y>
      <width>451</width>
      <height>51</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="editKey">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>110</y>
      <width>451</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelInput">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>67</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Input:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelKey">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>67</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Key:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelOutput">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>160</y>
      <width>67</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Output:</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="buttonEncrypt">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>40</y>
      <width>110</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Encrypt</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="buttonDecrypt">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>70</y>
      <width>110</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Decrypt</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>701</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
